tests/content_hash/test_content_hash.py

```diff
@@ -0,0 +1,46 @@
+from src.caselawclient.content_hash import (content_hash, hashable_text,
+                                            validate_content_hash)
+
+VALID_DOC = b"""<?xml version="1.0" encoding="UTF-8"?>
+    <akomaNtoso
+      xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
+      xmlns:uk="https://caselaw.nationalarchives.gov.uk/akn">
+      <judgment name="judgment">
+        <meta>
+        <proprietary>
+        <uk:hash>c4367ebc0937f4dc2d6b372d9a09670e3606a5b3da77a070149755db5f942565</uk:hash>
+        <uk:cite>the meta section should not be present BAD</uk:cite>
+        </proprietary>
+        </meta>
+        <p>Do <b>use</b></p>
+        <p>this <i>valid</i> text</p>
+      </judgment>
+    </akomaNtoso>
+    """
+
+INVALID_DOC = VALID_DOC.replace(b"Do", b"Do not").replace(b"valid", b"invalid")
+
+
+def test_hashable_text():
+    """Do we correctly identify the text to hash, omitting the meta section and removing spaces?"""
+    assert hashable_text(VALID_DOC) == b"Dousethisvalidtext"
```

Comment:
[non-blocking] I know the two cases (note my suggestion of removing the third) here are pretty simple, but I feel it could be worth splitting out the valid, invalid test cases into their own separate test functions, just so that if one of these test cases starts failing in the future we will know whether the positive or negative case has failed.

And similarly for the other tests

Not a blocker though